metc-wastewater-covid-monitor/server.R

```diff
@@ -342,7 +343,8 @@ server <- function(input, output) {
             size = 14,
             family = font_family_list,
             color = councilR::colors$suppBlack
-          )
+          ),
+          rangemode = "nonnegative"
```

Comment:
This appears to be accidentally added to the "title" structure, but it should be moved a few lines down into the "yaxis" structure instead.  In fact, you can see that this doesn't take effect if you visit the "COVID-19 VARIANTS" page, click on the 0 in the lower-left corner, and see that the minimum is -69.
https://metrotransitmn.shinyapps.io/metc-wastewater-covid-monitor/

Note that adding "nonnegative" will not improve readability on the variants page (the x-axis is still drawn at 0), but will only affect the amount of whitespace between the x-axis and the labels below.

Reply:
Nice catch! I'm surprised that plotly didn't throw a warning. I'll resolve this in a fresh pull request.